packages/volto/news/5554.documentation

```diff
@@ -0,0 +1 @@
+Mereged the index.md file of Contributing to Volto and Develop Volto Core ,so that there is a complete flow present for developers and deleted to Develop Volto Core File.@ujjwaleee26
```

Comment:
You have this inverted. Please see the issue #5556.

Reply:
@stevepiercy  okay ,sorry for this PR, i am doing all the corrections, and instead of yarn commands portion in `index.md` is should just give the link of `developing volto core`.Right? ,cause anyway. yarn commands doesn't work with core -volto, only suitable for Volto handson training

Reply:
Correct. Training is entirely separate from this issue.